packages/ai/src/ai/common/models/torch/torch_loader.py

```
@@ -0,0 +1,242 @@
+    def _get_memory_footprint(model: Any) -> float:
+        """Estimate GPU memory footprint from a loaded model."""
+        try:
+            from ai.common.torch import torch
+
+            if hasattr(model, 'parameters'):
+                total_params = sum(p.numel() for p in model.parameters())
+                bytes_per_param = 4
+                total_bytes = total_params * bytes_per_param * 1.2  # 20% overhead
+                return total_bytes / (1024 ** 3)
+        except Exception:
+            pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)